react/src/lib/components/SubsurfaceViewer/layers/polylines/polylinesLayer.ts

```diff
@@ -8,6 +8,9 @@ import {
     defineBoundingBox,
 } from "../utils/layerTools";
 
+type TIsPolylineClosedFunc = (index: number) => boolean;
+type TPathType = "open" | "loop" | null;
```

Comment:
Do we need the `null` option?

Reply:
This is Deck.GL PathLayer prop. If all the polylines are open or closed it can be set in PathLayer and it  will skip some checks for better performance.
If this prop is not set or null PathLayer checks closeness by comparing coordinates and decides how to draw the polylines.